### PR TITLE
Onl 5094 vision file

### PR DESCRIPTION
**Pull request by jaimeperezpedra**

### CODE REVIEW

File: VISION.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Vision
+
+Ebury Chameleon is developed thanks to the frontend expertise of the [Online team](https://fxsolutions.atlassian.net/wiki/spaces/TEAM/pages/119543461/ONL) members, Chameleon brings the components that Eburyonline needs to build the best online experience for the Ebury products.
```
Comment (stavros-tomas) [inline VISION.md:3]: Ebury labs link maybe? This Atlassian link won't work outside of the organisation

Comment (matallo) [reply VISION.md:3]: Minor thing, I prefer to keep this internal link.

File: VISION.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+The components from Chameleon are accessible through [https://chameleon.ebury.now.sh](https://chameleon.ebury.now.sh) and it's a live page when you can see all the components that the Online team has developed.
```
Comment (matallo) [inline VISION.md:5]:
```suggestion
The components from Chameleon are accessible through [https://chameleon.ebury.now.sh](https://chameleon.ebury.now.sh) and it's a live page where you can see all the components that the Online team has developed.
```

File: VISION.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+This library is written on [Vue.js](https://vuejs.org/) and the UI component explorer is [Storybook](https://storybook.js.org), we follow the Design system [Chameleon](https://github.com/Ebury/chameleon) with the help of utility-first styles library [Tailwind CSS](http://tailwindcss.com).
```
Comment (matallo) [inline VISION.md:7]:
```suggestion
This library is written in [Vue.js](https://vuejs.org/) and the UI components explorer is [Storybook](https://storybook.js.org), we follow the design guidelines from the UX/Design team with the help of utility-first styles library [Tailwind CSS](http://tailwindcss.com).
```

**Review by matallo — CHANGES_REQUESTED** [review]

**Review by stavros-tomas — APPROVED** [review]

File: VISION.md (hunk 1 above)
Comment (stavros-tomas) [reply VISION.md:3]: fair enough

**Review by matallo — APPROVED** [review]

**Review by jmesao — APPROVED** [review]

**Pull request merged**